versions.yaml

```diff
@@ -40,8 +62,6 @@ versions:
       - '1.15-debian9'
       - '1.15.12'
       - '1.15'
-      - 'mainline'
```

Comment:
Why are we removing mainline? How was it previously used?

Reply:
Initially I had added the version 1.17. But removed because version is not stable and forgot to re-add the mainline tag to 1.15. Just did it.